src/orquestra/sdk/_base/cli/_ui/_presenters.py

```diff
@@ -362,19 +362,51 @@ def show_wf_run(self, summary: ui_models.WFRunSummary):
         self._console.print(self.get_wf_run(summary))
 
     def show_wf_list(self, summary: ui_models.WFList):
+        """Display a list of workflow runs.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Nice docstring!